packages/loader/container-loader/src/loader.ts

```diff
@@ -108,6 +108,7 @@ export class RelativeLoader implements ILoader {
 }
 
 export interface ILoaderOptions extends ILoaderOptions1 {
+	// @deprecated - this option is no longer used
```

Comment:
@NicholasCouri Do you use this loader options for single commit summary?

Reply:
I don't suppose we do.